consensus/src/consensus.rs

```diff
@@ -264,6 +268,56 @@ impl<T: Operations + 'static, D: Database + 'static> Consensus<T, D> {
             Err(ConsensusError::MaxIterationReached)
         })
     }
+
+    async fn consensus_delay() {
+        let spin_time: u64 = env::var("RUSK_CONSENSUS_SPIN_TIME")
+            .unwrap_or_default()
+            .parse()
+            .unwrap_or_default();
+
+        if spin_time == 0 {
+            return;
+        }
+
+        info!("RUSK_CONSENSUS_SPIN_TIME is {spin_time}");
+
+        let spin_time = UNIX_EPOCH + Duration::from_secs(spin_time);
+        let mut now = SystemTime::now();
+        while spin_time > now {
+            let to_wait =
+                spin_time.duration_since(now).expect("When the hell am I?");
+
+            info!(
+                "Waiting {to_wait:?} for consensus to be triggered at {}",
+                time_util::print_system_time_to_rfc3339(&spin_time)
+            );
+
+            let chunk = match to_wait {
+                // More than 1h print every 15min
+                secs if secs > Duration::from_secs(60 * 60) => {
+                    Duration::from_secs(15 * 60)
+                }
+                // More than 30min print every 10in
+                secs if secs > Duration::from_secs(30 * 60) => {
+                    Duration::from_secs(10 * 60)
+                }
+                // More than 5min print every 5min
+                secs if secs > Duration::from_secs(5 * 60) => {
+                    Duration::from_secs(5 * 60)
+                }
+                // More than 1min print every 30secs
+                secs if secs > Duration::from_secs(60) => {
+                    Duration::from_secs(30)
+                }
+                // Countdown last minute
+                _ => Duration::from_secs(1),
+            };
+
+            tokio::time::sleep(chunk).await;
+            now = SystemTime::now();
+        }
+        env::remove_var("RUSK_CONSENSUS_SPIN_TIME");
+    }
```

Comment:
Do we really need this? We printed this because the network on ITN2 was stuck and needed to provide feedback to the user, but during normal operations this would just pollute the logs

Reply:
Mmm, on a second thought, this would not impact the logs as it is only for consensus spinning at a given time. Disregard